### ISSUE TITLE
No way to change the org affiliation or creating user for a framework 

### ISSUE THREAD

**Issue by jmarks** [opensalt/opensalt#279]
As a super admin I might want to change the owning org or user for a framework. This is not possible in the UI. At some point we might what multiple organizations to share a framework.  AS a work around, you can add users to the access list individually. Perhaps might also add org to the access list as  anew feature.

**Comment by brandonopened** [comment]
https://opensalt.mybalsamiq.com/projects/opensalt/edit/Org%20Detail . thoughts?

**Comment by jmarks** [comment]
I cant seem to see that. Can you post a screen grab?

**Comment by jmarks** [comment]
![image](https://user-images.githubusercontent.com/1319413/33636835-cdd9e67c-d9d2-11e7-8a2b-c00118bf1673.png)

**Comment by jmarks** [comment]
lets discuss in weekly meeting.

**Comment by brandonopened** [comment]
What might the scope be here?

**Comment by roverwolf** [comment]
Yes, @jmarks, lets break this up a bit.  The description adds quite a bit more the title indicates, each part with different priorities.

**Comment by jmarks** [comment]
THis is an issue for the CASE Network.
